src/hooks/queries/useEvmFees.ts

```diff
@@ -0,0 +1,68 @@
+import type { ChainId } from '@shapeshiftoss/caip'
+import { skipToken, useQuery } from '@tanstack/react-query'
+import { useMemo } from 'react'
+import { reactQueries } from 'react-queries'
+import { selectEvmFees } from 'react-queries/selectors'
+import { useWallet } from 'hooks/useWallet/useWallet'
+import {
+  assertGetEvmChainAdapter,
+  getFeesWithWallet,
+  type GetFeesWithWalletArgs,
+  isGetFeesWithWalletArgs,
+} from 'lib/utils/evm'
+import {
+  selectFeeAssetByChainId,
+  selectMarketDataByAssetIdUserCurrency,
+} from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+type UseEvmFeesProps = Omit<
+  GetFeesWithWalletArgs,
+  'wallet' | 'adapter' | 'accountNumber' | 'data'
+> & {
+  accountNumber: number | undefined
+  data: string | undefined
+  chainId: ChainId
+  enabled?: boolean
+  staleTime?: number
+  refetchInterval?: number | false | undefined
+  refetchIntervalInBackground?: boolean
+}
+
+export const useEvmFees = (props: UseEvmFeesProps) => {
```

Comment:
The only reason they're like so most of the time is because of monkey see monkey do, not conventions - not sure if we want a default vs. letting clients decide explicitly (or not, in which case we leverage the default `staleTime` and no `refetchInterval` / `refetchIntervalInBackground`

Reply:
As you want, stating that at the moment, all those props are always the same at 4 places and differ only at one, it might mean that we would like to set some defaults but we might also prefer to keep this definition at consumer level, happy with both

Reply:
Discussed this one out, agreed to keep it outside of the hook since:
1. This hook is only used in Arbitrum currently but will for other chains in other places in the future, and refetch intervals are obviously diff per chain/blocktime/context
2. The only reason why those are the same currently is because monkey see monkey do, but we probably want to change these depending on context in the future